---
content_type: page
description: This section provides the citations and links to the readings assigned
  for each session of the course.
learning_resource_types:
- Readings
ocw_type: CourseSection
title: Readings
uid: 1f3d1984-e0db-6e25-65c7-313934369c19
video_metadata:
  youtube_id: null
---

This textbook is recommended for the course:  
Zvelebil, Marketa J., and Jeremy O. Baum. _Understanding Bioinformatics_. Garland Science, 2007. ISBN: 9780815340249. \[Preview with {{% resource_link "834dc384-1fd1-4b19-b686-596f05df606a" "Google books" %}}\]

The instructors have also selected various texts as particularly useful in specific areas, if you are looking for more information. See the {{% resource_link fa95efc3-d3fe-cba9-9745-73504676f128 "textbook section on the syllabus" "#textbook" %}}.

{{< tableopen >}}
{{< theadopen >}}
{{< tropen >}}
{{< thopen >}}
LEC #
{{< thclose >}}
{{< thopen >}}
TOPICS
{{< thclose >}}
{{< thopen >}}
READINGS
{{< thclose >}}

{{< trclose >}}

{{< theadclose >}}
{{< tropen >}}
{{< tdopen >}}
1
{{< tdclose >}}
{{< tdopen >}}
Course Introduction; Overview
{{< tdclose >}}
{{< tdopen >}}
No readings for this lecture.
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
2
{{< tdclose >}}
{{< tdopen >}}
Local Alignment; Statistics
{{< tdclose >}}
{{< tdopen >}}


National Center for Biotechnology Information. "{{% resource_link "b15f978d-85a0-45ec-973a-3ad556414f5e" "The Statistics of Sequence Similarity Scores" %}}." BLAST Tutorial.

Metzker, Michael L. "{{% resource_link "9381a768-ead7-4902-8df7-cd29faa2cef1" "Sequencing Technologies—The Next Generation" %}}." _Nature Reviews Genetics_ 11, no. 1 (2010): 31–46.


{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
3
{{< tdclose >}}
{{< tdopen >}}
Global Alignment of Protein Statistics
{{< tdclose >}}
{{< tdopen >}}
No readings for this lecture.
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
4
{{< tdclose >}}
{{< tdopen >}}
Comparative Genomics
{{< tdclose >}}
{{< tdopen >}}


Sabeti, P. C., S. F. Schaffner, et al. "{{% resource_link "3dc2a14d-139a-490e-96c7-91d54f85b7c1" "Positive Natural Selection in the Human Lineage" %}}." _Science_ 312, no. 5780 (2006): 1614–20.  
Read the first three pages.

Bejerano, Gill, Michael Pheasant, et al. "{{% resource_link "ca54c280-184c-4672-83fc-1b75860e3c30" "Ultraconserved Elements in the Human Genome" %}}." _Science_ 304, no. 5675 (2004): 1321–5.

Pennacchio, Len A., Nadav Ahituv, et al. "{{% resource_link "20bae721-e31f-4033-a367-03f35dc4f14d" "In Vivo Enhancer Analysis of Human Conserved Non–coding Sequences" %}}." _Nature_ 444, no. 7118 (2006): 499–502.

Visel, Axel, Shyam Prabhakar, et al. "{{% resource_link "4fe47636-4185-4b49-b4fa-eef2fc490b2e" "Ultraconservation Identifies a Small Subset of Extremely Constrained Developmental Enhancers" %}}." _Nature Genetics_ 40, no. 2 (2008): 158–60.

Bejerano, Gill, Craig B. Lowe, et al. "{{% resource_link "6abd706b-2125-4373-a7af-d7932cc9d292" "A Distal Enhancer and an Ultraconserved Exon are Derived from a Novel Retroposon" %}}." _Nature_ 441, no. 7089 (2006): 87–90.

Lareau, Liana F., Maki Inada, et al. "{{% resource_link "56659ca2-4fe8-484e-a4ad-9e773856d145" "Unproductive Splicing of SR Genes Associated with Highly Conserved and Ultraconserved DNA Elements" %}}." _Nature_ 446, no. 7138 (2007): 926–9.

Lewis, Benjamin P., I–hung Shih, et al. "{{% resource_link "4335a8d9-ef80-43f7-880c-c936eff22104" "Prediction of Mammalian MicroRNA Targets" %}}." _Cell_ 115, no. 7 (2003): 787–98.

Lewis, Benjamin P., Christopher B. Burge, et al. "{{% resource_link "afffc506-900b-4a28-b1ed-b60bf854001b" "Conserved Seed Pairing, Often Flanked by Adenosines, Indicates that Thousands of Human Genes are MicroRNA Targets" %}}." _Cell_ 120, no. 1 (2005): 15–20.

Kheradpour, Pouya, Alexander Stark, et al. "{{% resource_link "e68c6113-fcc5-47f7-b45a-6dbdb8cc4635" "Reliable Prediction of Regulator Targets Using 12 Drosophila Genomes" %}}." _Genome Research_ 17, no. 12 (2007): 1919–31.

Friedman, Robin C., Kyle Kai–How Farh, et al. "{{% resource_link "d3b69d08-8692-4400-815a-61f93af8b171" "Most Mammalian mRNAs are Conserved Targets of MicroRNAs" %}}." _Genome Research_ 19, no. 1 (2009): 92–105.

Graveley, Brenton R. "{{% resource_link "0d82bd4b-dad7-40e8-a1f4-ae01f0da2f78" "Mutually Exclusive Splicing of the Insect Dscam Pre–mRNA Directed by Competing Intronic RNA Secondary Structures" %}}." _Cell_ 123, no. 1 (2005): 65–73.

Jansen, Ruud, Jan Embden, et al. "{{% resource_link "8e63b86a-5f63-4311-8e09-bda6214bb3bf" "Identification of Genes that are Associated with DNA Repeats in Prokaryotes" %}}." _Molecular Microbiology_ 43, no. 6 (2002): 1565–75.

Bolotin, Alexander, Benoit Quinquis, et al. "{{% resource_link "05ba5f7f-d104-4980-97d4-524ecac1d4f0" "Clustered Regularly Interspaced Short Palindrome Repeats (CRISPRs) have Spacers of Extrachromosomal Origin" %}}." _Microbiology_ 151, no. 8 (2005): 2551–61.


{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
5
{{< tdclose >}}
{{< tdopen >}}
Read Alignment
{{< tdclose >}}
{{< tdopen >}}


Langmead, Ben, Cole Trapnell, et al. "{{% resource_link "af4a617a-2175-426a-ad69-9d8ec8f99f1c" "Ultrafast and Memory–efficient Alignment of Short DNA Sequences to the Human Genome" %}}." _Genome Biology_ 10, no. 3 (2009): R25.

Li, Heng, and Richard Durbin. "{{% resource_link "35733683-e302-4fa4-9a5a-bcff662015e4" "Fast and Accurate Short Read Alignment with Burrows–wheeler Transform" %}}." _Bioinformatics_ 25, no. 14 (2009): 1754–60.

Trapnell, Cole, and Steven L. Salzberg. "{{% resource_link "f4710306-9d48-4ff7-80fa-f72ede89775d" "How to Map Billions of Short Reads onto Genomes" %}}." _Nature Biotechnology_ 27, no. 5 (2009): 455.

{{% resource_link "9f413cc3-3606-4119-973b-bb0ac7578c30" "Burrows–Wheeler Aligner" %}}

{{% resource_link "6e6206e8-91c6-4e4b-8790-1ea5f7d0e3e3" "Bowtie" %}}: An ultrafast memory–efficient short read aligner


{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
6
{{< tdclose >}}
{{< tdopen >}}
Genome Assembly
{{< tdclose >}}
{{< tdopen >}}


Simpson, Jared T., and Richard Durbin. "{{% resource_link "4d0846ac-12d0-43d4-9273-9a11049705bb" "Efficient De Novo Assembly of Large Genomes Using Compressed Data Structures" %}}." _Genome Research_ 22, no. 3 (2012): 549–56.

Zerbino, Daniel R., and Ewan Birney. "{{% resource_link "9fe53835-cf10-4a9b-91af-26000a53b15d" "Velvet: Algorithms for De Novo Short Read Assembly Using De Bruijn Graphs" %}}." _Genome Research_ 18, no. 5 (2008): 821–9.


{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
7
{{< tdclose >}}
{{< tdopen >}}
ChIP-seq / IDR
{{< tdclose >}}
{{< tdopen >}}


Guo, Yuchun, Georgios Papachristoudis, et al. "{{% resource_link "b1db235c-ea93-4f12-b808-99627d808d1d" "Discovering Homotypic Binding Events at High Spatial Resolution" %}}." _Bioinformatics_ 26, no. 24 (2010): 3028–34.

Li, Qunhua, James B. Brown, et al. "{{% resource_link "2feccf43-a227-42ea-b898-c6a62cf8aabd" "Measuring Reproducibility of High–throughput Experiments" %}}." _The Annals of Applied Statistics_ 5, no. 3 (2011): 1752–79.


{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
8
{{< tdclose >}}
{{< tdopen >}}
RNA–seq Analysis
{{< tdclose >}}
{{< tdopen >}}


Trapnell, Cole, Brian A. Williams, et al. "{{% resource_link "5dfd3640-224d-4f9a-ac2f-b5dd70ff4711" "Transcript Assembly and Quantification by RNA–seq Reveals Unannotated Transcripts and Isoform Switching during Cell Differentiation" %}}." _Nature Biotechnology_ 28, no. 5 (2010): 511–5.

Anders, Simon, and Wolfgang Huber. "{{% resource_link "4f5519b2-7b2c-4bd4-8b15-75c1a321e464" "Differential Expression Analysis for Sequence Count Data" %}}." _Genome Biology_ 11, no. 10 (2010): R106.  
Licensed under CC–BY.

Wang, Zhong, Mark Gerstein, et al. "{{% resource_link "3b48e966-5e2a-47bc-9202-62fd99324b91" "RNA–Seq: a Revolutionary Tool for Transcriptomics" %}}." _Nature Reviews Genetics_ 10, no. 1 (2009): 57–63.

Shalek, Alex K., Rahul Satija, et al. "{{% resource_link "2461cc50-f73a-40d2-96f4-34df64519d76" "Single–cell Transcriptomics Reveals Bimodality in Expression and Splicing in Immune Cells" %}}." _Nature_ 498 (2013): 236–40.

Smith, Lindsay I. {{% resource_link "3a1e0d21-d789-4dec-b3d3-03dfbbe317a0" "\"A Tutorial on Principal Components Analysis.\" (PDF)" %}} February 26, 2002.


{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
9
{{< tdclose >}}
{{< tdopen >}}
Modeling and Discovery of Sequence Motifs (Gibbs Sampler, Alternatives)
{{< tdclose >}}
{{< tdopen >}}


D'haeseleer, Patrik. "{{% resource_link "5a7b57a7-2f9f-4931-8b2f-442208ea489a" "What are DNA Sequence Motifs?" %}}" _Nature Biotechnology_ 24, no. 4 (2006): 423–25.

———. "{{% resource_link "2417aa21-9f08-4650-8a49-80118940fb4d" "How does DNA Sequence Motif Discovery Work?" %}}" _Nature Biotechnology_ 24, no. 8 (2006): 959–61.

Eddy, Sean R. "{{% resource_link "56f5d2c3-652c-441d-a42d-63c778901ccd" "What is Bayesian Statistics?" %}}" _Nature Biotechnology_ 22, no. 9 (2004): 1177–8.

Bailey, Timothy L., and Charles Elkan. "{{% resource_link "94817419-fd4a-48c5-8ae4-b950997864e5" "Unsupervised Learning of Multiple Motifs in Biopolymers Using Expectation Maximization" %}}." _Machine Learning_ 21, no. 1–2 (1995): 51–80.

Lawrence, Charles E., Stephen F. Altschul, et al. "{{% resource_link "f478cfcf-a943-48ae-96e5-636658b9c84e" "Detecting Subtle Sequence Signals: A Gibbs Sampling Strategy for Multiple Alignment" %}}." _Science_ 262, no. 5131 (1993): 208–14.


{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
10
{{< tdclose >}}
{{< tdopen >}}
Markov and Hidden Markov Models
{{< tdclose >}}
{{< tdopen >}}


Eddy, Sean R. "{{% resource_link "c4857649-effb-4107-95b6-ded5db297c63" "What is a Hidden Markov Model?" %}}" _Nature Biotechnology_ 22, no. 10 (2004): 1315–6.

Rabiner, Lawrence. "{{% resource_link "e62b02cb-d5df-4e98-81cf-81739cc00b5e" "A Tutorial on Hidden Markov Models and Selected Applications in Speech Recognition" %}}." _Proceedings of the IEEE_ 77, no. 2 (1989): 257–86.


{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
11
{{< tdclose >}}
{{< tdopen >}}
RNA Secondary Structure Prediction
{{< tdclose >}}
{{< tdopen >}}
Eddy, Sean R. "{{% resource_link "afeeab72-f3ff-4bcd-ab04-9c89c75eb29b" "How do RNA Folding Algorithms Work?" %}}" _Nature Biotechnology_ 22, no. 11 (2004): 1457–8.
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
12
{{< tdclose >}}
{{< tdopen >}}
Introduction to Protein Structure
{{< tdclose >}}
{{< tdopen >}}


Scheeff, Eric D., and J. Lynn Fink. "Fundamentals of Protein Structure." In _Structural Bioinformatics_. Edited by Philip E. Bourne and Helge Weissig. Wiley–Liss, 2003, pp. 15–39. \[Preview with {{% resource_link "33748607-464d-4068-9741-8f14bfe6f057" "Google Books" %}}\]


{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
13
{{< tdclose >}}
{{< tdopen >}}
Predicting Protein Structure
{{< tdclose >}}
{{< tdopen >}}
Moretti, Rocco, Sarel J. Fleishman, et al. "{{% resource_link "20ee1320-cf57-4a62-8b14-f39e56b90979" "Community‐wide Evaluation of Methods for Predicting the Effect of Mutations on Protein–protein Interactions" %}}." _Proteins: Structure, Function, and Bioinformatics_ 81, no. 11 (2013): 1980–7.
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
14
{{< tdclose >}}
{{< tdopen >}}
Predicting Interactions
{{< tdclose >}}
{{< tdopen >}}


Tuncbag, Nurcan, Attila Gursoy, et al. "{{% resource_link "7d1b17ec-d849-4da0-a0d4-c74313f8342c" "Predicting Protein–protein Interactions on a Proteome Scale by Matching Evolutionary and Structural Similarities at Interfaces Using PRISM" %}}." _Nature Protocols_ 6, no. 9 (2011): 1341–54.

Zhang, Qiangfeng Cliff, Donald Petrey, et al. "{{% resource_link "31a8890a-2334-49d3-bba8-cc98bcabd6e7" "Structure–based Prediction of Protein–protein Interactions on a Genome–wide Scale" %}}." _Nature_ 490, no. 7421 (2012): 556–60.

Jansen, Ronald, Haiyuan Yu, et al. "{{% resource_link "98c210d1-f94d-43ff-9b16-27a9e0de58b3" "A Bayesian Networks Approach for Predicting Protein–protein Interactions from Genomic Data" %}}." _Science_ 302, no. 5644 (2003): 449–53.


{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
15
{{< tdclose >}}
{{< tdopen >}}
Gene Regulatory Networks
{{< tdclose >}}
{{< tdopen >}}
Marbach, Daniel, James C. Costello, et al. "{{% resource_link "4572e7da-fe01-4c3b-834d-5574db344b31" "Wisdom of Crowds for Robust Gene Network Inference" %}}." _Nature Methods_ 9, no. 8 (2012): 796–804.
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
16
{{< tdclose >}}
{{< tdopen >}}
Protein Interaction Networks
{{< tdclose >}}
{{< tdopen >}}
No readings for this lecture.
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
17
{{< tdclose >}}
{{< tdopen >}}
Logic Modeling of Cell Signaling Networks. Guest Lecture: {{% resource_link "29efabc5-2add-4d86-ab1a-c2b338ab5e0f" "Doug Lauffenburger" %}}
{{< tdclose >}}
{{< tdopen >}}


Morris, Melody K., Julio Saez–Rodriguez, et al. "{{% resource_link "4c91f9bc-bd36-4ed4-86b4-27de83f60f89" "Logic–based Models for the Analysis of Cell Signaling Networks" %}}." _Biochemistry_ 49, no. 15 (2010): 3216–24.

Saez‐Rodriguez, Julio, Leonidas G. Alexopoulos, et al . "{{% resource_link "1543a0a7-1dad-4c02-b492-01b3fa65c511" "Discrete Logic Modelling as a Means to Link Protein Signalling Networks with Functional Analysis of Mammalian Signal Transduction" %}}." _Molecular Systems Biology_ 5, no. 1 (2009): 331.


{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
18
{{< tdclose >}}
{{< tdopen >}}
Analysis of Chromatin Structure
{{< tdclose >}}
{{< tdopen >}}


Hoffman, Michael M., Orion J. Buske, et al. "{{% resource_link "9d9284a3-64c8-4912-9970-6b8345d17946" "Unsupervised Pattern Discovery in Human Chromatin Structure through Genomic Segmentation" %}}." _Nature Methods_ 9, no. 5 (2012): 473–6.

Zhou, Vicky W., Alon Goren, et al. "{{% resource_link "acbe1657-2556-4b32-bdbd-7caec32c8e52" "Charting Histone Modifications and the Functional Organization of Mammalian Genomes" %}}." _Nature Reviews Genetics_ 12, no. 1 (2010): 7–18.

Sherwood, Richard I., Tatsunori Hashimoto, et al. "{{% resource_link "88f62b32-3bd4-46d0-8a7e-47d32f8beb85" "Discovery of Directional and Nondirectional Pioneer Transcription Factors by Modeling DNase Profile Magnitude and Shape" %}}." _Nature Biotechnology_ 32, no. 2 (2014): 171–8.

Dostie, Josée, and Job Dekker. "{{% resource_link "0f29926d-5ef5-4468-ba6b-3af4aa2f1d4b" "Mapping Networks of Physical Interactions between Genomic Elements Using 5C Technology" %}}." _Nature Protocols_ 2, no. 4 (2007): 988–1002.


{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
19
{{< tdclose >}}
{{< tdopen >}}
Discovering Quantitative Trait Loci (QTLs)
{{< tdclose >}}
{{< tdopen >}}


Bloom, Joshua S., Ian M. Ehrenreich, et al. "{{% resource_link "6b7b9c7b-71d4-4cbf-b154-48816257960e" "Finding the Sources of Missing Heritability in a Yeast Cross" %}}." _Nature_ 494, no. 7436 (2013): 234–7.

Broman, Karl W., and Saunak Sen. "Single–QTL Analysis." Chapter 4 in _A Guide to QTL Mapping with R/qtl_. Springer, 2009. ISBN: 9780387921242. \[Preview with {{% resource_link "d821a049-dfd1-4582-b106-0910a3cc3880" "Google Books" %}}\]


{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
20
{{< tdclose >}}
{{< tdopen >}}
Genome Wide Associate Studies
{{< tdclose >}}
{{< tdopen >}}


Li, Heng. "{{% resource_link "f4f842fa-14a9-468e-9a5b-4261ccd824e1" "A Statistical Framework for SNP Calling, Mutation Discovery, Association Mapping and Population Genetical Parameter Estimation from Sequencing Data" %}}." _Bioinformatics_ 27, no. 21 (2011): 2987–93.

Roberts, Nicholas J., Joshua T. Vogelstein, et al. "{{% resource_link "1e850c76-9e00-4de7-93e5-d29006188644" "The Predictive Capacity of Personal Genome Sequencing" %}}." _Science Translational Medicine_ 4, no. 133 (2012): 133ra58.

1000 Genomes. "{{% resource_link "6305d1f1-5d3e-4896-b6e8-a8ade6b79acf" "Variant Call Format" %}}."

Goldstein, David B., Andrew Allen, et al. "{{% resource_link "f89f56e5-52f6-4f75-8361-3990f40c0ab4" "Sequencing Studies in Human Genetics: Design and Interpretation" %}}." _Nature Reviews Genetics_ 14, no. 7 (2013): 460–70.


{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
21
{{< tdclose >}}
{{< tdopen >}}
Synthetic Biology: From Parts to Modules to Therapeutic Systems. Guest Lecture: {{% resource_link "7319cabe-2a21-4252-a12d-550550906fe9" "Ron Weiss" %}}
{{< tdclose >}}
{{< tdopen >}}
No readings for this lecture.
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
22
{{< tdclose >}}
{{< tdopen >}}
Causality, Natural Computing, and Engineering Genomes. Guest Lecture: {{% resource_link "bec02140-bec9-4cc1-b1e6-cc07b3fb4614" "George Church" %}}
{{< tdclose >}}
{{< tdopen >}}
No readings for this lecture.
{{< tdclose >}}

{{< trclose >}}

{{< tableclose >}}