---
content_type: page
description: This section contains links to tools and web resources that visitors
  may find helpful in their study of computational and systems biology.
learning_resource_types:
- Tools
ocw_type: CourseSection
title: Tools
uid: 4d56c5fd-ee8c-6f20-503a-a8eef0d89847
video_metadata:
  youtube_id: null
---

Computational Biology Tools and Resources
-----------------------------------------

{{< tableopen >}}
{{< theadopen >}}
{{< tropen >}}
{{< thopen >}}
TOOLS
{{< thclose >}}
{{< thopen >}}
DESCRIPTIONS
{{< thclose >}}
{{< thopen >}}
REFERENCES
{{< thclose >}}

{{< trclose >}}

{{< theadclose >}}
{{< tropen >}}
{{< tdopen >}}
{{% resource_link "ccf8ea8d-7c13-4a8b-9aa0-a7f6ed9e53d8" "BLAST" %}}
{{< tdclose >}}
{{< tdopen >}}
Sequence alignments provide a powerful way to compare novel sequences with previously characterized genes. Both functional and evolutionary information can be inferred from well designed queries and alignments. The Basic Local Alignment Search Tool (BLAST) provides a method for rapid searching of nucleotide and protein databases. It is a sequence comparison algorithm optimized for speed used to search sequence databases for optimal local alignments to a query.
{{< tdclose >}}
{{< tdopen >}}
Altschul, S. F., W. Gish, et al. "{{% resource_link "e595452d-f73e-4d77-8eb8-c5327f5c5408" "Basic Local Alignment Search Tool" %}}." _Journal of Molecular Biology_ 215, no. 3 (1990): 403–10.
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
{{% resource_link "b27e61fd-1741-4776-8fee-700186efd2be" "BIND" %}}
{{< tdclose >}}
{{< tdopen >}}
The Biomolecular Interaction Network Database (BIND) is a collection of records documenting molecular interactions. The contents of BIND include high-throughput data submissions and hand-curated information gathered from the scientific literature.
{{< tdclose >}}
{{< tdopen >}}
Bader, G. D., D. Betel, et al. "{{% resource_link "9bd648ff-0f94-4cba-9426-d37ded8efe92" "BIND: The Biomolecular Interaction Network Database" %}}." _Nucleic Acids Research_ 31, no. 1 (2003): 248–50.
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
{{% resource_link "851e877d-7012-47ab-b7bb-88bc0ca0b0dc" "Biology WorkBench" %}}
{{< tdclose >}}
{{< tdopen >}}
The Biology WorkBench is a web-based tool for biologists developed by the San Diego Supercomputer Center at the University of California San Diego. The WorkBench allows biologists to search many popular protein and nucleic acid sequence databases. Database searching is integrated with access to a wide variety of analysis and modeling tools, all within a point and click interface that eliminates file format compatibility problems.
{{< tdclose >}}
{{< tdopen >}}
 
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
{{% resource_link "424226cd-5429-4214-b881-8a4a8363a088" "ClustalW" %}}
{{< tdclose >}}
{{< tdopen >}}
ClustalW is a general purpose multiple sequence alignment program for DNA or proteins. It produces biologically meaningful multiple sequence alignments of divergent sequences.
{{< tdclose >}}
{{< tdopen >}}
Thompson, J. D., D. G. Higgins, et al. "{{% resource_link "f2befa0e-38fb-4b02-b7e2-77ad5c6299d9" "CLUSTAL W: Improving the Sensitivity of Progressive Multiple Sequence Alignment through Sequence Weighting, Position-specific Gap Penalties and Weight Matrix Choice" %}}." _Nucleic Acids Research_ 22, no. 22 (1994): 4673–80.
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
{{% resource_link "c7c19d4d-15b3-465c-b9cf-c71da202192a" "DALI" %}}
{{< tdclose >}}
{{< tdopen >}}
DALI stands for Distance mAtrix aLIgnment. The Dali server is an automatic service for the comparison of protein structure in 3D. You send the coordinates of a query structure and receive a multiple structure alignment in return. You can submit your coordinates either by electronic mail or interactively from the World Wide Web.
{{< tdclose >}}
{{< tdopen >}}
Holm, L., and C. Sander. "{{% resource_link "1ef4629a-b324-460b-8037-a18f9d7cfe2b" "Mapping the Protein Universe" %}}." _Science_ 273, no. 5275 (1996): 595–602.
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}


{{% resource_link "398f178f-8049-49f3-9f9c-04a13a903313" "Deep View Swiss-Pdb Viewer" %}}

{{% resource_link "e2d44e8d-564a-47d1-b88d-80df5b7699ea" "Tutorial For Deep View (Swiss-PdbViewer)" %}}


{{< tdclose >}}
{{< tdopen >}}
The Deep View Swiss-PdbViewer is a software application with a user friendly interface that allows one to analyze several proteins at the same time. The proteins can be superimposed in order to deduce structural alignments and compare their active sites or any other relevant parts. Amino acid mutations, H-bonds, angles, and distances between atoms are easy to obtain thanks to the intuitive graphic and menu interface.
{{< tdclose >}}
{{< tdopen >}}
Guex, N., and M. C. Peitsch. "{{% resource_link "d4e568ed-72d7-496e-b9f8-fd69e269c320" "SWISS-MODEL and the Swiss-PdbViewer: An Environment for Comparative Protein Modeling" %}}." _Electrophoresis_ 18, no. 15 (1997): 2714–23.
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
{{% resource_link "99f47b7f-5803-44c1-aa3d-dd1db761ddb2" "DIP" %}}{{< sup "TM" >}}
{{< tdclose >}}
{{< tdopen >}}
DIP{{< sup "TM" >}} stands for Database of Interacting Proteins. The DIP{{< sup "TM" >}} database catalogs experimentally determined interactions between proteins. It combines information from a variety of sources to create a single, consistent set of protein-protein interactions.
{{< tdclose >}}
{{< tdopen >}}
Salwinski L., C. S. Miller, et al. "{{% resource_link "7ef6c0cc-686b-4bdd-85e3-7657d0c8db87" "The Database of Interacting Proteins: 2004 Update" %}}." _Nucleic Acids Research_ 32, no. 1 (2004): D449–51.
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
Dot Matrix
{{< tdclose >}}
{{< tdopen >}}
Dot or matrix plots provide an easy and powerful means of sequence analysis for searching out regions of similarity in two sequences and repeats within a single sequence.
{{< tdclose >}}
{{< tdopen >}}


Maizel, J. V., and R. P. Lenk. "{{% resource_link "c17d1d0f-ab76-4538-a015-9898ff573a38" "Enhanced Graphic Matrix Analysis of Nucleic Acid and Protein Sequences" %}}." _Proceedings of the National Academy Sciences_ 78, no. 12 (1981): 7665–9.

Pustell, J., and F. C. Kafatos. "{{% resource_link "9cd5fe24-fb92-4eea-997b-d182c4f1b172" "A High Speed, High Capacity Homology Matrix: Zooming through SV40 and Polyoma" %}}." _Nucleic Acids Research_ 10, no. 15 (1982): 4765–82.

Quigley, G. J., L. Gehrke, et al. "{{% resource_link "43a63681-f203-4e78-b0be-7918d758e378" "Computer-aided Nucleic Acid Secondary Structure Modeling Incorporating Enzymatic Digestion Data" %}}." _Nucleic Acids Research_ 12, no. 1 (1984): 347–66.


{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}


{{% resource_link "4878e9e0-05ca-4307-acc3-416591372bd8" "Entrez" %}}

{{% resource_link "4ba6cdce-88a8-4106-a780-6ff21d67b2b4" "NCBI" %}}


{{< tdclose >}}
{{< tdopen >}}
Entrez is a retrieval system designed for searching several linked databases at NCBI (National Center for Biotechnology Information).
{{< tdclose >}}
{{< tdopen >}}
 
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
{{% resource_link "9864cca5-4c67-4d8f-a0a7-f4048cc1a20d" "GENSCAN" %}}
{{< tdclose >}}
{{< tdopen >}}
GENSCAN predicts the locations and exon-intron structures of genes in genomic sequences from a variety of organisms. GENSCAN was developed by Prof. Chris Burge while he was in the research group of Samuel Karlin, Department of Mathematics, Stanford University.
{{< tdclose >}}
{{< tdopen >}}
Burge, C., and S. Karlin. "{{% resource_link "4414b973-06c5-46eb-aff7-fe1097105626" "Prediction of Complete Gene Structures in Human Genomic DNA" %}}." _Journal of Molecular Biology_ 268, no. 1 (1997): 78–94.
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
{{% resource_link "f30a98f6-2a1a-473c-baa9-cfbc17f7946a" "Gibbs Motif Sampler" %}}
{{< tdclose >}}
{{< tdopen >}}
The Gibbs Motif Sampler will allow you to identify motifs, conserved regions, in DNA or protein sequences.
{{< tdclose >}}
{{< tdopen >}}
This software was developed by Eric C. Rouchka and Bill Thompson based on work by C. E. Lawrence, J. S. Liu, A. F. Neuwald and others as part of the Bayesian Bioinformatics Program at the Biometrics Laboratory of Wadsworth Center.
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
{{% resource_link "38938924-4542-4af1-bcc3-5cd23af5f19e" "MEME" %}}
{{< tdclose >}}
{{< tdopen >}}
Discover motifs (highly conserved regions) in groups of related DNA or protein sequences using MEME.
{{< tdclose >}}
{{< tdopen >}}
Bailey, Timothy L., and Charles Elkan. "{{% resource_link "1cb7977f-9000-4063-a9f7-10a996bec568" "Fitting a Mixture Model by Expectation Maximization to Discover Motifs in Biopolymers" %}}." _Proceedings of the Second International Conference on Intelligent Systems for Molecular Biology_ 2 (1994): 28–36.
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
MODBASE
{{< tdclose >}}
{{< tdopen >}}
MODBASE is a database of annotated comparative protein structure models and associated resources.
{{< tdclose >}}
{{< tdopen >}}
Pieper U., N. Eswar, et al. "{{% resource_link "cf738242-fcae-458a-aa19-b430eaa90fed" "MODBASE: A Database of Annotated Comparative Protein Structure Models, and Associated Resources" %}}." _Nucleic Acids Research_ 34, no. 1 (2004): D291–5.
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}


{{% resource_link "5fed2769-1f81-44c2-8bb0-3f1c98017844" "PDB   " %}}


{{< tdclose >}}
{{< tdopen >}}
The Protein Data Bank (PDB) is the single worldwide repository for the processing and distribution of 3-D biological macromolecular structure data.
{{< tdclose >}}
{{< tdopen >}}
Berman, H. M., J. Westbrook, et al. "{{% resource_link "ca0612f5-cd65-40b4-8ef9-567addec9b7b" "The Protein Data Bank" %}}." _Nucleic Acids Research_ 28, no. 1 (2000): 235–42.
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
{{% resource_link "a5b65e28-7bb9-4f8f-83f7-4fed85c065ae" "PHYLIP" %}}
{{< tdclose >}}
{{< tdopen >}}
PHYLIP (the PHYLogeny Inference Package) is a package of programs for inferring phylogenies (evolutionary trees). It is available free over the Internet, and written to work on as many different kinds of computer systems as possible.
{{< tdclose >}}
{{< tdopen >}}
 
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
{{% resource_link "4ba6cdce-88a8-4106-a780-6ff21d67b2b4" "NCBI" %}}
{{< tdclose >}}
{{< tdopen >}}
Established in 1988 as a national resource for molecular biology information, NCBI (National Center for Biotechnology Information) creates public databases, conducts research in computational biology, develops software tools for analyzing genome data, and disseminates biomedical information—all for the better understanding of molecular processes affecting human health and disease. This web site provides access to a myriad of biological sequence databases, structural databases, bioinformatics tools, and literature search tools.
{{< tdclose >}}
{{< tdopen >}}
 
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
{{% resource_link "77c54f92-8c60-478b-93d3-88ef90b58390" "Python Scripting Language" %}}
{{< tdclose >}}
{{< tdopen >}}
A scripting language widely used, along with PERL, for bioinformatics and computational biology.
{{< tdclose >}}
{{< tdopen >}}
 
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
{{% resource_link "ae9373c9-9d52-42f7-bffc-bc78f6de9bcd" "RasMol" %}}
{{< tdclose >}}
{{< tdopen >}}
RasMol is a molecular graphics program intended for the visualization of proteins, nucleic acids and small molecules. The program reads in molecular co-ordinate files and interactively displays the molecule on the screen in a variety of representations and colour schemes.
{{< tdclose >}}
{{< tdopen >}}
Sayle R., and E. James Milner-White. "{{% resource_link "b5f68c0a-cd7c-41c0-850a-d613740e4c57" "RasMol: Biomolecular Graphics for All" %}}." _Trends in Biochemical Sciences_ 20, no. 9 (1995): 374–6.
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
{{% resource_link "7254640a-e16d-4cb9-8673-e3df665fbf04" "Scansite" %}}
{{< tdclose >}}
{{< tdopen >}}
Scansite searches for motifs within proteins that are likely to be phosphorylated by specific protein kinases or bind to domains such as SH2 domains, 14-3-3 domains or PDZ domains.
{{< tdclose >}}
{{< tdopen >}}


Songyang, Z., S. Blechner, et al. "{{% resource_link "96b07606-453a-432f-be5a-da27e587ed9e" "Use of an Oriented Peptide Library to Determine the Optimal Substrates of Protein Kinases" %}}." _Current Biology_ 4, no. 11 (1994): 973–82.

Yaffe, M. B., G. G. Leparc, et al. "{{% resource_link "ae30c571-ea7a-4dc3-bfb2-ff917cead14b" "A Motif-based Profile Scanning Approach for Genome-wide Prediction of Signaling Pathways" %}}." _Nature Biotechnology_ 19, no. 4 (2001): 348–53.

Obenauer, J. C., L. C. Cantley, et al. "{{% resource_link "8828bbce-fe8a-4227-b48f-eb80b87196ff" "Scansite 2.0: Proteome-wide Prediction of Cell Signaling Interactions using Short Sequence Motifs" %}}." _Nucleic Acids Research_ 31, no. 13 (2003): 3635–41.


{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
{{% resource_link "305a5767-7879-440e-8f04-f0870f7808ca" "SCOP" %}}
{{< tdclose >}}
{{< tdopen >}}
Nearly all proteins have structural similarities with other proteins and, in some of these cases, share a common evolutionary origin. The Structural Classification Of Proteins (SCOP) database, created by manual inspection and abetted by a battery of automated methods, aims to provide a detailed and comprehensive description of the structural and evolutionary relationships between all proteins whose structure is known. As such, it provides a broad survey of all known protein folds, detailed information about the close relatives of any particular protein, and a framework for future research and classification.
{{< tdclose >}}
{{< tdopen >}}
Murzin, A. G., S. E. Brenner, et al. "{{% resource_link "a811b4ac-b42a-41d0-80ba-30d431f2ca13" "SCOP: A Structural Classification of Proteins Database for the Investigation of Sequences and Structures" %}}." _Journal of Molecular Biology_ 247, no. 4 (1995): 536–40.
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
{{% resource_link "29d2ef3e-5187-4e98-89cd-b04476ce99b2" "TMHMM" %}}
{{< tdclose >}}
{{< tdopen >}}
This software tool was developed by the Center for Biological Sequence Analysis at the Technical University of Denmark and is used to predict transmembrane helices in protein sequences.
{{< tdclose >}}
{{< tdopen >}}
Krogh, A., B. Larsson, B., et al. "{{% resource_link "978b5fcb-5866-4c6c-bbf1-c08c50befc5c" "Predicting Transmembrane Protein Topology with a Hidden Markov Model: Application to Complete Genomes" %}}." _Journal of Molecular Biology_ 305, no. 3 (2001): 567–80.
{{< tdclose >}}

{{< trclose >}}

{{< tableclose >}}